setup.cfg

```
@@ -1,7 +1,7 @@
 [metadata]
 name=benchcab
 summary= Software to run a benchmarking suite for CABLE LSM
-version=2.0.0
+# version=2.0.0
```

```suggestion
version=2.0.0
```
Purely for having cleaner commit history when we deal with #129

---

Definitely, leftover to be removed. Thanks